packages/yew/src/backend/web_sys.rs

```diff
@@ -0,0 +1,62 @@
+use crate::backend::RenderingBackend;
+use anyhow::{anyhow, Error};
+use web_sys::{
+    Document, Element, HtmlButtonElement as ButtonElement, HtmlInputElement as InputElement,
+    HtmlTextAreaElement as TextAreaElement, InputEvent, Node, Text as TextNode, Window,
+};
+
+/// The rendering backend for web_sys
+#[derive(Debug)]
+pub struct Renderer {}
+
+impl RenderingBackend for Renderer {
+    type Element = Element;
+    type Node = Node;
+    type Document = Document;
+    type Window = Window;
+    type InputEvent = InputEvent;
+
+    type InputElement = InputElement;
+    type ButtonElement = ButtonElement;
+    type TextAreaElement = TextAreaElement;
+    type TextNode = TextNode;
+
+    /// Returns the `host` for the current document. Useful for connecting to the server which serves
+    /// the app.
+    fn get_host() -> Result<String, Error> {
```

Comment:
Might it be better to use `this_error` here for a more ergonomic interface?
```suggestion
    fn get_host() -> Result<String, Error> {
```